theme/brb/static/images/logo/cloud-transparent-mono-long.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
svg images are accepted widely: https://caniuse.com/#feat=svg but we should at least enable gzip compression